scripts/check_team_pickers.py

```
@@ -0,0 +1,245 @@
+            where = "{}: {}".format(plugin, league)
+
+            espn_path = LEAGUE_PATHS.get(league)
+            if not espn_path:
+                problems.append("{} - unknown league key; add it to LEAGUE_PATHS"
+                                .format(where))
+                continue
+
+            if espn_path not in rosters:
+                try:
+                    rosters[espn_path] = fetch_teams(espn_path)
+                except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)